src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  Alphabet,
+  Length,
+  Default,
+}
+
+function getSortedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+): string[] {
+  const sortedGoodsCopy = [...goods];
+
+  if (sortField === SortType.Alphabet) {
+    sortedGoodsCopy.sort((good1, good2) => {
+      return good1.localeCompare(good2);
+    });
+  }
+
+  if (sortField === SortType.Length) {
```

Comment:
Would be great to use switch...case here as you have a limited amount of sortFields